src/docfx/localization/LocalizationConvention.cs

```diff
@@ -122,6 +120,52 @@ public static bool TryGetContributionBranch(string branch, out string contributi
             return false;
         }
 
+        public static (Error error, string content, Document file) TryResolveFromHistory(Docset docset, string pathToDocset)
+        {
+            // try to resolve from source repo's git history
+            // todo: support code snippet
+            var fallbackDocset = GetFallbackDocset();
+            if (fallbackDocset != null && Document.GetContentType(pathToDocset) == ContentType.Page)
```

Comment:
To be consistent, this feature is about fallback to git history for all _content inclusions_, including TOC content inclusion, but right now TOC loader does not respect `content` returned from `ResolveContent`, it will trigger the `Debug.Assert(!IsFromHistory);` above: https://github.com/dotnet/docfx/blob/v3/src/docfx/build/toc/parser/TableOfContentsParser.cs#L198

Strictly speaking, `TocHrefType.RelativeFolder` is not a content inclusion.